### ISSUE TITLE
Is IPE still under development? 

### ISSUE THREAD

**Issue by brianjoubert** [otfried/ipe#172]
Hello,

I have to say that I am an IPE fan for a few years now. I make all my figures and presentation with it. I wanted to know if this project is still under development concerning new features or more importantly for me performance enhancement?

Cheers

**Comment by otfried** [comment]
Yes, Ipe is under active development.   A new version will probably be released during the winter break.

Feel free to suggest features or to point out where performance is insufficient (in that case make sure to attach example files to reproduce the behaviour).

**Comment by brianjoubert** [comment]
Thank you for your reply! Concerning new features, for me it would be nice
to have an embedded color gradient option. Also a linestyle that would look
like a pencil trace could be nice. I tried to convert some non geek friends
to ipe that got reluctant because they had to use a terminal for pdftoipe.
Maybe adding a "convert" option in ipe directly could help.

Concerning the performance, when I have a presentation that starts to be a
bit big (80 slides) The software freezes during about 2-3 seconds when I
save the document and even if I just added a tiny line. I guess it
generates the full pdf everytime you save.

Congratulations again for this software!
